config/manager/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: aws/cloudwatch-agent-operator
-  newTag: 1.1.0
+  newTag: 1.1.1
```

Comment:
I wonder if this should be bumped to 1.2.0 since its a major feature launch, feel free to disagree

Reply:
You are right, i am changing minor version now rather than patch version.